mathics/builtin/linalg.py

```diff
@@ -693,42 +699,81 @@ def apply(self, m, evaluation):
 class Eigenvalues(Builtin):
     """
     <dl>
-    <dt>'Eigenvalues[$m$]'
-        <dd>computes the eigenvalues of the matrix $m$.
+      <dt>'Eigenvalues[$m$]'
+      <dd>computes the eigenvalues of the matrix $m$.
+      By default Sympy's routine is used. Sometiems this is slow and
+      less good than the corresponding mpmath routine. Use option Method->"mpmath" if you want
+      to use mpmath's routine instead.
     </dl>
 
-    >> Eigenvalues[{{1, 1, 0}, {1, 0, 1}, {0, 1, 1}}] // Sort
-     = {-1, 1, 2}
+    Numeric eigenvalues are sorted in order of decreasing absolute value:
+    >> Eigenvalues[{{1, 1, 0}, {1, 0, 1}, {0, 1, 1}}]
+     = {2, -1, 1}
 
+    Symbolic eigenvalues:
     >> Eigenvalues[{{Cos[theta],Sin[theta],0},{-Sin[theta],Cos[theta],0},{0,0,1}}] // Sort
      = {1, Cos[theta] + Sqrt[(-1 + Cos[theta]) (1 + Cos[theta])], Cos[theta] - Sqrt[(-1 + Cos[theta]) (1 + Cos[theta])]}
 
     >> Eigenvalues[{{7, 1}, {-4, 3}}]
      = {5, 5}
 
+    >> Eigenvalues[{{7, 1}, {-4, 3}}]
+     = {5, 5}
+
     #> Eigenvalues[{{1, 0}, {0}}]
      : Argument {{1, 0}, {0}} at position 1 is not a non-empty rectangular matrix.
      = Eigenvalues[{{1, 0}, {0}}]
+
+    Compute Eigenvalues using mpmath's routines; Sympy is slow here and returns
+    complex numbers.
+    >> Eigenvalues[{{-8, 12, 4}, {12, -20, 0}, {4, 0, -2}}, Method->"mpmath"]
```

Comment:
I got this idea from WL's [Eigenvector](https://reference.wolfram.com/language/ref/Eigenvalues.html). It uses strings for implementation options. I was thinking of "mpmath" more akin to "Arnoldi" ,"Banded", or "FEAST" rather than Automatic.

When I grep for "Method" right now in mathics/builtin, I am not coming up with _any_ instances where there is a Symbol used, just strings: 

```
image.py:    >> Threshold[img, Method -> "Mean"]
image.py:    >> Threshold[img, Method -> "Median"]
image.py:    options = {"Method": '"Cluster"'}
lists.py:        "bdmtd": 'Method in `` must be either "Optimize", "Agglomerate" or "KMeans".',
```

Reply:
> I got this idea from WL's [Eigenvector](https://reference.wolfram.com/language/ref/Eigenvalues.html). It uses strings for implementation options. I was thinking of "mpmath" more akin to "Arnoldi" ,"Banded", or "FEAST" rather than Automatic.
> 
> When I grep for "Method" right now in mathics/builtin, I am not coming up with _any_ instances where there is a Symbol used, just strings:
> 
> ```
> image.py:    >> Threshold[img, Method -> "Mean"]
> image.py:    >> Threshold[img, Method -> "Median"]
> image.py:    options = {"Method": '"Cluster"'}
> lists.py:        "bdmtd": 'Method in `` must be either "Optimize", "Agglomerate" or "KMeans".',
> ```

My bad, strings are the right choice then.